### ISSUE TITLE
Sorting Tablist

### ISSUE THREAD

**Issue by charlie-works** [montlikadani/TabList#260]
### Description

This most likely is already part of the plugin, it's just I can't get it to work. If it already is part of the plugin, can you explain how to turn it on?

**Comment by TheNety** [comment]
If you haven't already created groups in groups.yml, then create them - [https://github.com/montlikadani/TabList/wiki/How-to-create-a-group](url) and then just add sort-priority to each group like in the example below:

```
groups:
  owner:
    prefix: '[Owner]'
    tabname: ''
    suffix: ''
    sort-priority: 99
  builder:
    prefix: '[Builder]'
    tabname: ''
    suffix: ''
    sort-priority: 98
```

**Comment by charlie-works** [comment]
so this wouldn't work?

groups:
  Owner:
    sort-priority: 6
  Moderator:
    sort-priority: 5
  Sponsor+:
    sort-priority: 4
  Sponsor:
    sort-priority: 3
  Supporter:
    sort-priority: 2
  Member:
    sort-priority: 1

**Comment by charlie-works** [comment]
does prefix have to be the same as the prefix for the rank? using another rank plugin. thanks!

**Comment by TheNety** [comment]
Prefix can be whatever you want, even a placeholder.

**Comment by charlie-works** [comment]
Let me rephrase.

I am using another plugin that assigns ranks to players, and I would like those ranks to be sorted in the tablist, will the above work?

By the way, thank you very much for the support!

**Comment by TheNety** [comment]
Should work for any vault (permission plugin) group if you add it to TabList's groups.yml. You're welcome!
